explore_utils.py

```diff
@@ -101,3 +101,14 @@ def find_channel_grouping_revenue(dataset):
     means = {grouping: np.mean(df[df['channelGrouping'] == grouping]['totals.transactionRevenue'].astype('int64')) / 10000 for grouping in groupings}
 
     return counts, means
+
+def find_percentage_single_transaction(dataset):
+
+    train_df2 = dataset.train
+    train_df2.set_index('fullVisitorId', inplace=True)
```

Comment:
Please make a shallow copy of the dataframe before changing the index. Otherwise, this change will be reflected in `explore.py` for everyone. See the comments I made in PR #49 for more information on how to fix.